app/assets/stylesheets/partials/anime-page.css.scss

```diff
@@ -519,6 +519,8 @@
       margin-bottom: 35px;
       padding: 25px;
       padding-bottom: 0;
+      display: inline-flex;
```

Comment:
I remember having trouble with flexbox and our prefixer not playing along back with the chat.  I'd double check that this works in all browsers and has the necessary prefixes.

Also flexbox stronk